siteflags/models.py

```diff
@@ -314,6 +314,9 @@ def is_flagged(self, user: 'User' = None, *, status: int = None) -> int:
         :param status: Optional status filter
 
         """
+        if user and not user.id:
```

Comment:
`is_anonymous` check would be more obvious

Reply:
I used `not user.id` because that was the one used for `set_flag`.